apps/docs/components/docs/components/helper.ts

```
@@ -0,0 +1,187 @@
+        });
+
+        startToken.types = ["folderStart"];
+        (startToken as TransformTokensTypes).folderContent = folder;
+        (startToken as TransformTokensTypes).summaryContent = [
+          ...token,
+          ellipsisToken,
+          copyContent,
+          ...endToken,
+        ];
+        (startToken as TransformTokensTypes).index = index;
+        isShowFolder && !fold && ((startToken as TransformTokensTypes).open = true);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)